Practice2.java

```diff
@@ -0,0 +1,29 @@
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Practice2 {
+	public static void main(String[] args) throws IOException {
+
+		System.out.println("1 ~ 999 ������ �ܼ� �Է��ϼ��� : "); // �Է��϶�� ����
+		BufferedReader input = new BufferedReader(new InputStreamReader(System.in)); // �Է¹ޱ�
+		String num = input.readLine(); // ���ڿ��� �Է°� �ޱ�
+		int inputNum = Integer.parseInt(num); // �޾� �� ���ڿ�� ���� ��ȯ�� ��� ����� �ޱ�
+
+		if (inputNum > 999) {
```

Comment:
이 부분에서 논리연산자 OR 을 쓰시면 else if 없이 코드를 작성할 수 있습니다! 

```suggestion
		if (inputNum > 999 || inputNum < 1) 
```
요런식으로요~

Reply:
넵! 적용해보겠습니다